packages/fluentui/accessibility/src/behaviors/Button/buttonGroupBehavior.ts

```diff
@@ -0,0 +1,18 @@
+import { Accessibility } from '../../types';
+
+/**
+ * @specification
+ * Adds role 'group' to 'root' slot.
+ */
+
+const buttonGroupBehavior: Accessibility<ButtonGroupBehaviorProps> = props => ({
+  attributes: {
+    root: {
+      role: 'group',
```

Comment:
Please could you check with @kolaps33 if that role is needed for the button group?